packages/react/src/stories/ic-pagination.stories.mdx

```diff
@@ -0,0 +1,125 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  ArgsTable,
+  Description,
+} from "@storybook/addon-docs";
+
+import { IcPagination, IcPaginationItem } from "../components";
+import readme from "../../../web-components/src/components/ic-pagination/readme.md";
+
+<Meta title="React Components/Pagination" component={IcPagination} />
+
+<Description markdown={readme} />
+
+### Simple
+
+<Canvas>
+  <Story name="Simple">
+  <p>Default/Dark/Light appearance</p>
+    <IcPagination pages="15" />
+    <br/>
+    <IcPagination pages="20" appearance="dark" />
+    <br/>
+    <div style={{ backgroundColor: "#2c2f34",  width: "fit-content", padding: "6px 8px" }}>
+      <IcPagination pages="15" appearance="light" />
+    </div>
+    <br/>
+  <p>Hidden current page</p>
+    <IcPagination pages="15" hideCurrentPage />
+    <br/>
+  <p>Hidden first and last buttons</p>
+    <IcPagination pages="15" hideFirstAndLastPageButton />
+    <br/>
+  <p>Default page</p>
+    <IcPagination pages="15" defaultPage="3" />
+    <br/>
+  <p>Disabled</p>
+    <IcPagination pages="15" defaultPage="3" disabled />
+    <br/>
+  <p>Label set</p>
+    <IcPagination pages="15" label="Label" />
+  </Story>
+</Canvas>
+
+### Complex
+
+<Canvas>
+  <Story name="Complex">
+  <p>Default/Dark/Light appearance</p>
+    <IcPagination pages="15" type="complex" />
+    <IcPagination pages="15" type="complex" appearance="dark" />
+    <div style={{ backgroundColor: "#2c2f34",  width: "fit-content", padding: "6px 8px" }}>
+      <IcPagination pages="15" type="complex" appearance="light" />
+    </div>
+    <br/>
+  <p>Hidden first and last buttons</p>
+    <IcPagination pages="15" type="complex" hideFirstAndLastPageButton />
+    <br/>
+  <p>Default page</p>
+    <IcPagination pages="15" type="complex" defaultPage={7} />
+    <br/>
+  <p>Disabled</p>
+    <IcPagination pages="15" type="complex" disabled />
+  </Story>
+</Canvas>
+
+### Complex - Adjacent & Boundary Items Set
+
+<Canvas>
+  <Story name="Complex - Adjacent & Boundary Items Set">
+    <div>
+      <p style={{ padding: 10 }}>Adacent: 0, Boundary: 0</p>
```

Comment:
```suggestion
      <p style={{ padding: 10 }}>Adjacent: 0, Boundary: 0</p>
```
I think this line has been copy and pasted for the rest so will need updating for those too